detection_rules/kbwrap.py

```diff
@@ -71,6 +72,36 @@ def upload_rule(ctx, rules, replace_id):
     return results
 
 
+@kibana_group.command('import-rules')
```

Comment:
Will this deprecate the `@root.command('import-rules')` command?

Reply:
no but we may want to consider renaming these to differentiate - good call